README.md

```diff
@@ -165,4 +158,13 @@ This project is licensed under the Apache 2.0 license - see the
 Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on the process for
 contributing to this project.
 
-Be mindful of our [Code of Conduct](CODE_OF_CONDUCT.md).
\ No newline at end of file
+Be mindful of our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Legacy
```

Comment:
This should not be the last section. It should be above the Breaking Changes section.